contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -75,7 +83,9 @@
 
   private Row currentRow;
 
-  private Workbook workbook;
+  private StreamingWorkbook swb;
+
+  private CoreProperties fileMetadata;
```

Comment:
This field is obtained from the previous one. Are there any reasons for adding it?

Reply:
Yeah, so both the workbook and the `fileMetadata` fields are called a few times later in the reader.  It seemed to make sense to create class variables for them.

Reply:
It is called in a single method only. May be declared a local variable in this method instead of adding the class field.

Reply:
Refactored with `fileMetadata` as local variable.